app-typescript/components/TabList.tsx

```diff
@@ -32,14 +32,22 @@ class TabList extends React.PureComponent<ITabList, IState> {
             index: 0,
         };
         this.handleChange = this.handleChange.bind(this);
+        this.goTo = this.goTo.bind(this);
     }
 
-    handleChange(index: number) {
+    private handleChange(index: number) {
         this.setState({
             index: index,
         });
     }
 
+    goTo(label: string) {
```

Comment:
add a `public` keyword so it's clear for other people reading the component that it's meant to be exposed.